var/spack/repos/builtin/packages/py-typish/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyTypish(PythonPackage):
+    """Python library for type checking
+
+    * Functions for thorough checks on types
+    * Instance checks considering generics
+    * Typesafe Duck-typing
```

Comment:
`spack info py-typish` removes newline characters so the formatting of this will be messed up. I would either simplify the docstring or add commas/semi-colons.